fossunited/chapters/doctype/foss_chapter_event/foss_chapter_event.py

```diff
@@ -211,6 +211,7 @@ def get_speakers(self):
         )
         speakers = []
         for cfp in speaker_cfps:
+            user = frappe.get_doc(USER_PROFILE, {"email": cfp.submitted_by})
```

Comment:
see line 218 - `user.profile_photo` is used but `user` isn't defined. this is the one real issue that was caught by introducing the `F` flag